.circleci/config.yml

```diff
@@ -6,9 +6,6 @@ orbs:
 executors:
   cross-builder:
     docker:
-      # NOTE: To upgrade the Go version, first push the upgrade to the cross-builder Dockerfile in the edge repo,
```

Comment:
Some version of this comment might still be useful

Reply:
Restored the relevant bit